game/graphics/opengl_renderer/OpenGLRenderer.cpp

```diff
@@ -74,11 +74,14 @@ OpenGLRenderer::OpenGLRenderer(std::shared_ptr<TexturePool> texture_pool,
       m_version(version) {
   // setup OpenGL errors
   glEnable(GL_DEBUG_OUTPUT);
+  // requires OpenGL 4.3
+#ifndef __APPLE__
   glDebugMessageCallback(opengl_error_callback, nullptr);
   // disable specific errors
   const GLuint gl_error_ignores_api_other[1] = {0x20071};
   glDebugMessageControl(GL_DEBUG_SOURCE_API, GL_DEBUG_TYPE_OTHER, GL_DONT_CARE, 1,
                         &gl_error_ignores_api_other[0], GL_FALSE);
```

Comment:
These are the reason I didn't just go with `SDL_GL_CONTEXT_MINOR_VERSION, 1` for all platforms. Not using two different versions, but only 4.1 for all platforms with OpenGL, I think would make things simpler. I don't think a debugging callback at all is available in 4.1 though.

Reply:
I agree. This debugging callback is super useful, so it's worth the effort to keep for linux/windows.